ex4/ex4.c

```diff
@@ -9,7 +9,22 @@
 
 int main(int argc, char* argv[])
 {
-    // Your code here    
+    int rc = fork();
+
+    if (rc < 0) 
+    {
+      fprintf(stderr, "Nobody here but us chickens...\n");
+      exit(1);
+    }
+    else if (rc == 0)
+    {
+        printf("Child process\n");
```

Comment:
Keep your spacing consistent here. You've used two spaces up to this point, so try to keep it consistent throughout.